core-bundle/contao/themes/flexible/styles/main.css

```diff
@@ -54,8 +54,11 @@ html {
 
 /* Body */
 body {
+	display: flex;
+	flex-direction: column;
 	background: var(--body-bg);
-	overflow-y: scroll;
+	min-height: 100svh;
+	/*overflow-y: scroll;*/
 }
```

Comment:
I think I’d go with grid layout here and avoid the `svh` unit, like this:

```css
html, body {
	height: 100%;
}
body {
	display: grid;
	grid-template-rows: auto 1fr;
	background: var(--body-bg);
	overflow-y: scroll;
}
```
and remove `min-height` and `flex-grow` from the `#container`

This could be adjusted later on to use subgrid for example so I think this is the best way forward.
But it’s also just an opinion, so feel free to keep the flexbox if you prefer that ☺️

Reply:
I'll try that one out!

Reply:
Introduced in 8ab086e213ba6503e0bb6fcc9b025394ba10c15c @ausi

Reply:
Should now use grid with your hints you added on slack :)